File: sample-videochat-webrtc/sample-videochat-webrtc/Components/InfoScreen/InfoTableViewCell/InfoTableViewCell.m
//
//  InfoTableViewCell.m
//  sample-videochat-webrtc
//
//  Created by Injoit on 3/12/19.
//  Copyright © 2019 Quickblox. All rights reserved.
//

#import "InfoTableViewCell.h"

@implementation InfoTableViewCell

- (void)applyInfo:(InfoModel*)model
{
    self.titleInfoLabel.text = model.title;
    self.descriptInfoLabel.text = model.info;
}

@end

File: sample-videochat-webrtc-swift/sample-videochat-webrtc-swift/Components/InfoScreenViewController/InfoTableViewCell.swift
//
//  InfoTableViewCell.swift
//  sample-videochat-webrtc-swift
//
//  Created by Injoit on 12/30/18.
//  Copyright © 2018 QuickBlox. All rights reserved.
//

import UIKit

class InfoTableViewCell: UITableViewCell {

    @IBOutlet weak var titleInfoLabel: UILabel!
    @IBOutlet weak var descriptInfoLabel: UILabel!
    
    func applyInfo(model: InfoModel) {
        titleInfoLabel.text = model.title
        descriptInfoLabel.text = model.info
    }
}
